CHANGELOG.md

```diff
@@ -3,3 +3,5 @@
    instead of using Thread.raise on the plugins' threads. Ref: https://github.com/elastic/logstash/pull/3895
  - Dependency on logstash-core update to 2.0
  - Update to allow semicolons (or not, e.g. Zabbix 2.0). #6 (davmrtl)
+ - Update to prevent Logstash from crashing when the Zabbix server becomes unavailable while the plugin is sending data. 
+   Ref: https://github.com/logstash-plugins/logstash-output-zabbix/pull/9
```

Comment:
When the PR is from the same repository, you can actually just use the `#9` syntax, like the `#6` above.  Github translates it for you.